config/config.go

```diff
@@ -179,6 +180,11 @@ func (e *Entry) Set(key, value string) error {
 			value = "screwdrivercd/launcher"
 		}
 		e.Launcher.Image = value
+	case "uuid":
```

Comment:
I should probably add this to the [README](https://github.com/screwdriver-cd/sd-local/blob/master/README.md).

Reply:
add to help output and README